setup.py

```diff
@@ -27,7 +27,7 @@
       url='https://github.com/LANL-Seismoacoustics/pisces',
       download_url='https://github.com/LANL-Seismoacoustics/pisces/tarball/0.3.2',
       keywords = ['seismology', 'geophysics', 'database'],
-      install_requires=['numpy','obspy>=1.0','sqlalchemy>=1.0','Click'],
+      install_requires=['numpy','obspy>=1.0','sqlalchemy=1.3.24','Click'],
```

Comment:
Can we just do `'sqlalchemy >1.0, <1.4'` here instead of requiring one specific version?

Reply:
yes, I just didn't know what range of sqlalchemy would work.